flink-runtime/src/test/java/org/apache/flink/runtime/io/network/netty/NettyMessageSerializationTest.java

```diff
@@ -135,14 +148,17 @@ public void testEncodeDecode() {
 		}
 	}
 
-	private void testEncodeDecodeBuffer(boolean testReadOnlyBuffer) {
-		NetworkBuffer buffer = new NetworkBuffer(MemorySegmentFactory.allocateUnpooledSegment(1024), FreeingBufferRecycler.INSTANCE);
+	private void testEncodeDecodeBuffer(boolean testReadOnlyBuffer, boolean testCompressedBuffer) {
+		NetworkBuffer buffer = new NetworkBuffer(MemorySegmentFactory.allocateUnpooledSegment(BUFFER_SIZE), FreeingBufferRecycler.INSTANCE);
 
-		for (int i = 0; i < 1024; i += 4) {
-			buffer.writeInt(i);
+		for (int i = 0; i < BUFFER_SIZE; i += 8) {
+			buffer.writeLong(i);
```

Comment:
Are there any considerations for changing to `Long` instead of previous `Int`?

Reply:
if int is used, the data size become lager after compress.